standards/nextjs_standards.md

```diff
@@ -0,0 +1,95 @@
+# Next.js Standards
+
+# Style Guide
+
+This style guide provides a comprehensive overview of best practices, coding standards, and recommendations for working with Next.js. It aims to help developers write clean, efficient, and maintainable code.
+
+## General Principles
+
+- **Consistency**: Ensure consistency across your codebase in formatting, naming conventions, and file structure.
+- **Readability**: Write code that is easy to read and understand, with meaningful variable and function names.
+- **Simplicity**: Keep solutions simple and avoid unnecessary complexity.
+- **Performance**: Optimize for speed and efficiency, considering the performance implications of your code.
+
+## Writing
+
+- Write clear, concise sentences and avoid tangents.
+- Use simple words; for example, use "use" instead of "utilize".
+- Use an active voice for readability.
+- Avoid subjective words like "easy", "quick", "simple", "just", etc.
+- Write in the second person (you/your) and use gender-neutral language.
+- Ensure code examples are properly formatted and working2.
+
+## Code Formatting
+
+- Use Prettier for automatic code formatting to maintain consistency.
+- Configure ESLint with Next.js specific rules to catch common issues and enforce style guidelines.
+- Adopt a consistent indentation method (e.g., 2 spaces) across the project.
```

Comment:
Let's also add that we prefer airbnb style guide

Reply:
Got this 👌🏼